app/serializers/concerns/cached_serializer.rb

```diff
@@ -0,0 +1,17 @@
+module CachedSerializer
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def as_json(model, context)
+      if Panoptes.flipper["cached_serializer"].enabled?
+        cache_key = "#{model.class.to_s}/#{model.id}/#{model.updated_at.to_i}/context-#{Digest::MD5.hexdigest(context.to_json)}"
```

Comment:
any reason not to create a smaller cache key from using the hexdigest of the whole string?